actionview/test/activerecord/multifetch_cache_test.rb

```diff
@@ -18,8 +18,8 @@ def view_cache_dependencies
         []
       end
 
-      def combined_fragment_cache_key(key)
-        [ :views, key ]
+      def combined_fragment_cache_key(key = nil)
+        [ :views, key ].compact
```

Comment:
@schneems I have a benchmark that shows that `[ ].compact` for 2 items where the last might be nil is slower than checking for nil before appending the key, e.g.

```
array = [ :views ]
array << key if key
array
```

Is twice as fast as `.compact` if key is nil, and about 50% faster if key is not nil.

Here's a gist with the benchmark code and results at the bottom: https://gist.github.com/tomfakes/7dfb9cbd64d130aff54363146b3c50ad